_posts/2016/2016-05-26-magdeburger-developer-days.md

```diff
@@ -0,0 +1,78 @@
+Bernd Gronostay, who works closely together with the author of "Java ist auch ein Insel", talked about a lot of new features and API changes.
+Not only did he talk about the modifications, but also showed a lot of examples which illustrated these features very well. 
+The perhaps most interesting part in this session were the various show cases of lambda functions and streams.
+This enables developers to write Java in a functional syntax.
+
+{% image blog/blog-magdeburg-5.jpg %}
+
+After lunch there was a talk with a topic which was completely new to us:
+Functional Domain Driven Design (DDD) with F# held by Marco Heimeshoff. 
+He is one of the cofounders of the [DDD community](http://dddcommunity.org) in Germany.
+His introduction showed a new idea of how the software models should look like.
+The mapping between the Requirement Space (Domain) and the Implementation Space (Code) is usually a gap that needs to be bridged.
+The difficulty is to have the code representing exactly the real domain.
+But often developers have to check that there is no possibility to represent an illegal state.
+The simple solution with DDD is: Make illegal states unrepresentable!
+This is only a little extract of this session with a lot of new, interesting ideas and patterns for solving things in a deterministically different way that makes intense unit testing obsolete.
+
+{% image blog/blog-magdeburg-6.jpg %}
+
+The next session was an introduction into the JavaScript framework [React](https://facebook.github.io/react) by Oliver Zeigermann, who is the author of several JavaScript books.
+He showed and explained the principals of React.
+One feature of React is that it reperesents only the View in a Model-View-Controller (MVC) model. 
+So it is typical to have a mixture of JavaScript and HTML code in one file.
+Another feature is that the Document Object Model (DOM) tree is virtualized.
+That means, when a webpage flow is establihed only the objects that have change are updated via the calculation of a `diff` between the webpage states.
+The interesting thing about React is, that due to that technique websites are always very fast.
+
+{% image blog/blog-magdeburg-7.jpg %}
+
+The last session of the day was "Architekturdokumentation leicht gemacht" (architecture documentation for dummies) by Andreas Richter, the organizer of the [Softwerkskammer Magdeburg](https://www.softwerkskammer.org/groups/magdeburg).
+He introduced the [arc42](http://arc42.org) template and its most important parts. 
+The main benefits of such an architectural templates is that there is always the same structure.
+
+## Summary
+
+The Magdeburger Developer Days offered a lot of very interesting talks in a broad range of topics.
+We were stunned from the professionel level that orga team could establish. 
+The speakers were well choosen and the quality of talks was absolutely extraordinary.
+Overall, the experience of the conference was very great and a visit next year is definitely a reasonable.
```

Comment:
Remove "overall" at the beginning of the sentence.

...and a visit next year is definitely a reasonable.
> nd a visit next year is definitely reasonable